source/incidents-and-alerts/index.html.md.erb

```diff
@@ -19,7 +19,7 @@ Emails are sent to the GovWifi developers mailbox. Notifications are sent to the
 
 ### AWS Cloudwatch alerts
 
-Emails are sent to GovWifi critical alerts mailbox. Notifications the #govwifi-monitoring Slack channel.
+Emails are sent to GovWifi critical alerts mailbox. Notifications are sent to the #govwifi-monitoring Slack channel.
```

Comment:
I'd squash this change in to the first commit, since that's where the words are omitted.

Reply:
I can see where you're coming from. Let's have a chat about approaches to squashing commits next time we pair.